net/tshttpproxy/tshttpproxy_synology_test.go

```diff
@@ -41,7 +41,7 @@ func TestSynologyProxyFromConfigCached(t *testing.T) {
 			t.Fatalf("got %s, %v; want nil, nil", val, err)
 		}
 
-		if got, want := cache.updated, time.Unix(0, 0); got != want {
+		if got, want := cache.updated, time.Unix(0, 0); !got.Equal(want) {
 			t.Fatalf("got %s, want %s", got, want)
```

Comment:
if this fails, the local PDT or whatever time from time.Unix(0, 0) will probably be misleading

I'd prefer to normalize the timezone in the got/want variables first and then == would be fine (if the lint tool knew that, but it won't) just for the failure output to be nice. But Equal is fine.

Reply:
converted both values to UTC